.gitignore

```diff
@@ -18,11 +18,14 @@ dist/
 htmlcov/
 py3*/
 pyb/
-python_pylint_venv/
```

Comment:
Keep it? Some people may still have it locally.

Reply:
I didn't delete it, just moved a few lines below.